tabbed-mode-explainer.md

```diff
@@ -20,22 +20,21 @@ Add a new manifest field `tab_strip` which allows apps to customize the tab stri
 
 ```
 "tab_strip": {
-    "home_tab": "auto" | "absent" | {
-        "url": "auto" | <url>,
-        "icons": "auto" | [...],
+    "home_tab": "absent" | {
+        "icons": [...],
+        "scope_patterns": [{"pathname": "..."}]
     },
-    "new_tab_button": "auto" | "absent" | {
-        "url": "auto" | <url>,
-        "icons": "auto" | [...],
+    "new_tab_button": "absent" | {
+        "url": <url>,
     },
 },
 ```
 
-The home tab is a pinned tab that, if enabled for an app, should always be present when the app is open. This tab should never navigate - links clicked from this tab should open in a new app tab. Apps can choose to customize the URL this tab is locked to and the icon displayed on the tab.
+The home tab is a pinned tab that, if enabled for an app, should always be present when the app is open. This tab should not navigate outside of the home tab scope (as specified by `scope_patterns`), so navigation outside of this scope should open in a new tab. If the `home_tab` field is unset it will default to `absent`. Apps can choose to customize the icon displayed on the tab.
```

Comment:
```suggestion
The home tab is a pinned tab that, if enabled for an app, should always be present when the app is open. This tab should not navigate outside of the home tab scope (as specified by `scope_patterns`), so navigation outside of this scope should open in a new tab. If the `home_tab` field is unset, it will default to `absent`. Apps can choose to customize the icon displayed on the tab.
```